### DISCUSSION TITLE
Osu laser not running at all, windows 7 64 bit

### DISCUSSION THREAD

**Discussion by ballroomblitz** [ppy/osu#15370] [Q&A] [answered]
Hi, I really want to play Osu!laser but it will not run on my computer. When I click the program icon, it doesn't do anything and doesn't open at all. 
I can run osu stable just fine.

I'm not very tech savvy so please answer with easy language/baby steps! Thank you.

**Comment by Joehuu** [comment, marked as answer]
If you look at the readme, lazer doesn't support windows 7.

**Reply by frenzibyte** [reply]
Keep in mind that this is not intended to be a final decision, but it cannot be supported for the time being, unfortunately.

**Reply by peppy** [reply]
Realm isn't planning on adding windows 7 support, so it will likely be permanent.

**Reply by ballroomblitz** [reply]
Oh no! That really sucks as I was looking forward to trying out the new game modes. Will you eventually incorporate them into stable? I despise Win 10 with every fiber of my being so I refuse to "upgrade".  I didn't get a readme when I installed it, or I can't find it.

Edit: There seems to be some old threads on here that say some people got it to work on 7 with some tweaking (thats the part I didn't understand well). Is there an older build that supports it?

**Reply by frenzibyte** [reply]
By "readme", we refer to [this](https://github.com/ppy/osu#readme) ([this part in particular](https://github.com/ppy/osu#running-osu)).


> Edit: There seems to be some old threads on here that say some people got it to work on 7 with some tweaking (thats the part I didn't understand well). Is there an older build that supports it?

I'm not sure what this "tweaking" is, but it's likely to be modifying the source code by removing Realm (the database backend) or otherwise, and therefore not be any beneficial for you, unless you don't intend on connecting to the servers and submitting scores.

**Reply by bdach** [reply]
> therefore not be any beneficial for you, unless you don't intend on connecting to the servers and submitting scores.

this is putting it mildly, given that we're migrating everything to realm. *the game will no longer be able to function properly* without realm once that's completed.

**Reply by ballroomblitz** [reply]
> 
> 
> By "readme", we refer to [this](https://github.com/ppy/osu#readme) ([this part in particular](https://github.com/ppy/osu#running-osu)).
> 
> > Edit: There seems to be some old threads on here that say some people got it to work on 7 with some tweaking (thats the part I didn't understand well). Is there an older build that supports it?
> 
> I'm not sure what this "tweaking" is, but it's likely to be modifying the source code by removing Realm (the database backend) or otherwise, and therefore not be any beneficial for you, unless you don't intend on connecting to the servers and submitting scores.

I wouldn't mind not being able to connect to the server as at this point I really just want to play Lazer for the new game modes/rulesets and continue to use Stable for everything else. 

The user who ran Lazer on Win 7 appears to have downloaded some update and done something else I don't quite understand to get it to run, there's an old thread about it somewhere on here. I'm not techy enough to understand it.

**Reply by peppy** [reply]
The only way is to use an old version, sorry.

**Comment by dragondemon111311** [comment]
this is why i need to update windows 10

**Comment by dragondemon111311** [comment]
> Hi, I really want to play Osu!laser but it will not run on my computer. When I click the program icon, it doesn't do anything and doesn't open at all. I can run osu stable just fine.
> 
> I'm not very tech savvy so please answer with easy language/baby steps! Thank you.

same with me
@@
i think i know why im upgrade to windows 10 for @@
